test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,59 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
```

Comment:
add import to chai-subset:
```
chai.use(require('chai-subset'));
```

Reply:
i already done this, but i got the next error: 

![image](https://user-images.githubusercontent.com/100431019/176777578-d4570d79-383c-4c36-a029-7d6b98bbfcaf.png)